streaming/index.js

```diff
@@ -215,9 +209,9 @@ if (cluster.isMaster) {
   };
 
   const errorMiddleware = (err, req, res, next) => {
-    log.error(req.requestId, err);
+    log.error(req.requestId, err.toString());
```

Comment:
This *should* only print the error message, not the whole stack. And it does so on my dev environment. However, in production, it still dumps the entire stack trace into the logs...